cwms-data-api/src/main/java/cwms/cda/data/dto/location/kind/Outlet.java

```diff
@@ -11,58 +24,64 @@
 import com.fasterxml.jackson.databind.PropertyNamingStrategies;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonNaming;
-import cwms.cda.data.dto.Location;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.api.errors.RequiredFieldException;
 import cwms.cda.data.dto.CwmsId;
+import cwms.cda.data.dto.Location;
 import cwms.cda.formatters.Formats;
 import cwms.cda.formatters.annotations.FormattableWith;
 import cwms.cda.formatters.json.JsonV1;
+import java.util.List;
 
 @FormattableWith(contentType = Formats.JSONV1, formatter = JsonV1.class, aliases = {Formats.DEFAULT, Formats.JSON})
 @JsonDeserialize(builder = Outlet.Builder.class)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
-public class Outlet extends ProjectStructure
-{
-	private final CharacteristicRef characteristicRef;
+public class Outlet extends ProjectStructure {
+    private final CwmsId characteristicRef;
```

Comment:
with the name change, what do you think about `characteristicId` instead?

Reply:
It's a key to the properties table right?  Is that an id or a ref?  I'm not opposed either way tbh.

Reply:
If we're doing that distinction, I'd say it's a "properties ref" given the underlying structure is using properties, but a "characteristic id" since it represents a single "Characteristic". Unless that should be plural characteristics....

There aren't any other "ref" fields that I could find, so I would lean towards "id" for consistency.

Reply:
Alright, characteristicsId it is, cause you can have multiple characteristics in the properties if I recall.

Reply:
Would it instead be better to just include "The Characteristic"?

Reply:
We could certainly go out and retrieve the properties from the db.  I don't know that Outlets will have a significant amount of data in those.  I wasn't sure if that's something we wanted to include, since it's not always necessary information, and the hec-cwms-data-access JDBC counterpart is currently not retrieving that information, just the ref.

Reply:
@MikeNeilson is that something you'd like us to do?

Reply:
Not a hard requirement, but as previously discussed it can be useful to just retrieve and thus de-normalize all data, especially if rather small. Which if these are just simple "properties" I imagine they would be.

Reply:
@MikeNeilson I spoke with @adamkorynta and for this PR I'm going to leave it with just the id.  We realized if we return the properties, we'd also have to decide on how the post/put/patch logic will apply to it.
Adam also recommended that I speak with @krowvin about it, since this may be data he requires, and if it is, we can identify the requirements and discuss behaviors further.  I'll post that in our discussion about gate changes. #317